thop/utils.py

```diff
@@ -1,10 +1,11 @@
-
-def clever_format(num, format="%.2f"):
+def clever_format(num, precision=3):
     if num > 1e12:
-        return format % (num / 1e12) + "T"
+        return f"{num / 1e12 :.{precision}}T"
```

Comment:
"%.2f" gives more flexibility to control the print style.

Reply:
Emmmm, that is right.